packages/shared/friends/sagas.ts

```diff
@@ -1141,11 +1160,15 @@ function* handleSendPrivateMessage(action: SendPrivateMessage) {
 }
 
 function* handleUpdateFriendship({ payload, meta }: UpdateFriendship) {
-  const { action, userId } = payload
+  const { action, userId, future } = payload
 
   const client: SocialAPI | undefined = yield select(getSocialClient)
 
+  // Get feature flag value
+  const newFriendRequestFlow = isNewFriendRequestEnabled()
+
   if (!client) {
+    yield call(future.resolve, { userId: userId, error: FriendshipErrorCode.FEC_UNKNOWN })
```

Comment:
Simplify here and below.
```suggestion
    yield call(future.resolve, { userId, error: FriendshipErrorCode.FEC_UNKNOWN })
```